sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/java/com/azure/ai/inkrecognizer/InkRecognizerAsyncClient.java

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
From other existing Cognitive Service SDKs. They are using below license info, I think we should keep same license title for all cognitive service SDKs. 

```
/**
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License. See License.txt in the project root for
 * license information.
 */

```

Reply:
Fixed.